samcli/lib/package/ecr_uploader.py

```diff
@@ -133,8 +133,10 @@ def delete_artifact(self, image_uri: str, resource_id: str, property_name: str):
 
         except botocore.exceptions.ClientError as ex:
             # Handle Client errors such as RepositoryNotFoundException or InvalidParameterException
-            LOG.error("DeleteArtifactFailedError Exception : %s", str(ex))
-            raise DeleteArtifactFailedError(resource_id=resource_id, property_name=property_name, ex=ex) from ex
+            if "RepositoryNotFoundException" not in str(ex):
```

Comment:
what should be done in the else, I think just logging the exception is not enough

Reply:
I can add an echo statement to let the user know about this.

Reply:
my comment is not about writing some info to the customer, my question is shall SAM CLI fail in this case, and if yes, it should fail using some custom user exception, and if not, so will notifying the customer be enough, and what is the message we need to give it to the customer

Reply:
sorry @hnnasit .. it is my bad, I missed the `not` in the condition.